brainstormforce/astra
# AST-1556 - Extended ast-sortable control to make use of adding sub-controls into it

### Description
- Currently sortable added for single post structure
- Added clone, parent, tracker atts for sortable sub controls
- **{Do not merge this}**  after review & approval we need to remove dummy single post strcture's clonning option, after that we can merge it.
- **Not syncing it** because single post structure is updated here with dummy controls.
- Dependent PR to merge first  -https://github.com/brainstormforce/astra/pull/4141
- Reference Gist prepared, for future help - https://gist.github.com/imnavanath/1e387d2c8cf33c906a9d6160c7e1d54b

### Screenshots
- https://share.bsf.io/NQuxA1jO

### Types of changes
- New implementation

### How has this been tested?
- Check clonning works correctly
- Check subcontrols save their values correctly (after refresh)
- Right now added clone limit as 3 so till 3 clonner item should disable

### Checklist:
